test/test_nn.py

```diff
@@ -1976,6 +1976,7 @@ def test_Conv2d_inconsistent_types_on_GPU_with_cudnn(self):
         # but it should work with the same type
         nn.functional.conv2d(inputs.float(), weights.float(), bias.float())
 
+    @unittest.skipIf(IS_WINDOWS, 'TODO: fix this test for Windows')
```

Comment:
This test seems to be stuck when running on Windows

Reply:
Have you added it to the issue?

Reply:
Added to https://github.com/pytorch/pytorch/issues/4092